packages/app/src/debug/DebugFailedTest.cy.tsx

```diff
@@ -0,0 +1,40 @@
+import DebugFailedTest from './DebugFailedTest.vue'
+
+describe('<DebugFailedTest/>', () => {
+  it('mounts correctly', () => {
+    const testResult = {
+      id: '676df87878',
+      titleParts: ['Login', 'Should redirect unauthenticated user to signin page'],
+    }
+
+    cy.mount(() => (
+      <DebugFailedTest failedTestResult={testResult} />
+    ))
+
+    cy.findByTestId('test-row').children().should('have.length', 2).then((ele) => {
+      expect(ele).to.have.css('color', 'rgb(90, 95, 122)')
+    })
+
+    testResult.titleParts.forEach((title, index) => {
+      cy.findByTestId(`titleParts-${index}`).should('have.text', ` ${title }`)
```

Comment:
Looks like extra spaces are getting added to the output of the title parts, is this for spacing/padding? Should use CSS for that

Reply:
Great catch! Turns out that the `<span>` containing the chevron and title part was adding a whitespace to the text inside it. I changed this component from a `span` to a `div` which alleviates the issue